test/test_helper.rb

```diff
@@ -26,6 +26,10 @@
 require "active_model"
 require "action_controller"
 
+# TODO: Remove warning gem and the following lines when freerange/mocha#593 will be fixed
```

Comment:
Is https://github.com/freerange/mocha/issues/593 the correct issue to refer to?

Reply:
Yes it is, thank you.